### ISSUE TITLE
Can I have the code for how the .emoji works essentially?

### ISSUE THREAD

**Issue by ghost** [MonsterMannen/DiscordBotNodeJs#10]
I want to add that command to my bot and i'm not sure how i'd acomplish this.

If you wanna hmu on dcord

**Comment by MonsterMannen** [comment]
https://github.com/MonsterMannen/DiscordBotNodeJs/blob/master/commands/emoji.js

install the 3 packages that are required, copypaste the code and see if that works.

**Comment by ghost** [comment]
Nope. I've tried it. I made an emoji.js in my folder and it won't work as a reference and I tried just copy + pasting it in my command with the correct packages. Didn't seem to work tho.

**Comment by ghost** [comment]
index.js




emoji.js

**Comment by MonsterMannen** [comment]
Dont you need to require emoji.js (you didnt add .js) ?

Also what is this:
  emojiwall.wall();

.wall? No function like that in your emoji.js
